library/modules/es7.promise.finally.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+// https://github.com/tc39/proposal-promise-finally
+var $export = require('./_export');
+
+$export($export.P, 'Promise', {
+  'finally': function _finally(onFinally){
```

Comment:
Use `'finally': function(onFinally)`. For ES6+ engines it will create the correct `.name`, for old engines we can't do anything.